aws/shared-state/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    aws = {
+      version = "~> 4.0"
+    }
+  }
+}
+
+resource "aws_s3_bucket" "state" {}
+
```

Comment:
maybe add:
- default encryption (AES-256)
- default region (eu-west as preferred in the main.tf code)

Reply:
Will add encryption, add the region here as that should be set at provider level. Adding it there